mars/utils.py

```diff
@@ -1629,6 +1634,18 @@ def cache_tileables(*tileables):
             t.cache = True
 
 
+def report_event(severity, label, message):
```

Comment:
Seems a ray-specific function, put into ray backend.

Reply:
moved to mars/oscar/backends/ray/utils.py